src/etc/inc/ipsec.inc

```diff
@@ -920,7 +920,7 @@ function ipsec_get_loglevels() {
 	return $levels;
 }
 
-function ipsec_vti($ph1ent, $returnaddresses = false) {
+function ipsec_vti($ph1ent, $returnaddresses = false, $disabled = false) {
```

Comment:
This seems like it's misnamed, shouldn't it be `$skipdisabled` or something similar? Just calling it `$disabled` and then passing it a static value during validation seems awkward/confusing.

Reply:
ok, changed